### PR TITLE
refactor(template-generator): Create a unified output

### PR DESCRIPTION
**Pull request by mathias-vandaele**
Create an unified  to have a consistent data format and unified output

## Description

We had two different definition of the same class `OperationParseResult`.

This record has been moved to the `http-dsl` package and is now shared for both project `open-api` parser and `postman-collections-parser`

## Related issues

closes https://github.com/camunda/connectors/issues/2693

### CODE REVIEW

**Review by ev-codes — COMMENTED** [review]
Thanks for this, looks good!

One small thing: Could you edit the PR title and description to say "create a unified output" for easier look-up later?

**Comment by mathias-vandaele** [comment]
> Thanks for this, looks good!
> 
> One small thing: Could you edit the PR title and description to say "create a unified output" for easier look-up later?

Doing it right now!

**Review by johnBgood — APPROVED** [review]

File: element-template-generator/http-dsl/src/main/java/io/camunda/connector/generator/dsl/http/OperationParseResult.java [hunk 1]
```diff
@@ -14,12 +14,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.camunda.connector.generator.openapi;
+package io.camunda.connector.generator.dsl.http;
```
Comment (ev-codes) [inline element-template-generator/http-dsl/src/main/java/io/camunda/connector/generator/dsl/http/OperationParseResult.java:17]: One last thing - the PR title should follow Conventional Commit formatting

(I see how my original comment might have been confusing - my intended point was to add a noun after "unified" 😄 )

Comment (mathias-vandaele) [reply element-template-generator/http-dsl/src/main/java/io/camunda/connector/generator/dsl/http/OperationParseResult.java:17]: Oh yeah sorry, I didnt know the PR name should also follow this pattern doing it right now 😅

**Review by ev-codes — APPROVED** [review]

**Pull request merged**